CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## unreleased
 
+### Added
+
+- `InlineKeyboardButton::{pay, login, web_app, callback_game, pay}` constructors ([#231][pr231])
+
+### Changed
+
+-  `InlineKeyboardButtonKind::Pay`'s only field now has type `True` ([#231][pr231])
+
+### Removed
+
+- `InlineKeyboardButton::{text, kind}` functions ([#231][pr231])
```

Comment:
Maybe just deprecate them instead of hard removing?

Reply:
IMO they are useless (you need to set both kind and text on creation), but I'm fine with deprecating them

Reply:
I'm just afraid that someone's already using these functions, I guess it's better to deprecate them.

Reply:
Sure 👍🏻

Reply:
Done